test/salt/pillar/default.sls

```diff
@@ -69,3 +69,4 @@ proftpd:
   Modules:
     mod_radius: false
     mod_ident: true
+    mod_sftp: true
```

Comment:
This makes the `sftp` module be activated by default, which is not supposed to be I think

Reply:
It's for test, I will delete it !

Reply:
for tests, as said elsewhere, you need to create a specific suite in kitchen.yml, for automated tests not manual ones